certbot/certbot/_internal/cli/__init__.py

```diff
@@ -95,6 +95,11 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
         action="store_true", default=flag_default("preconfigured_renewal"),
         help=argparse.SUPPRESS
     )
+    helpful.add(
+        None, "--issuance-timeout", type=int,
```

Comment:
Let's restrict this to positive values:

```suggestion
        None, "--issuance-timeout", type=nonnegative_int,
```